test/fairplay.test.js

```diff
@@ -104,6 +119,8 @@ QUnit.test('lifecycle', function(assert) {
     assert.equal(callCounts.createSession, 1, 'a key session has been created');
     assert.equal(callCounts.getLicense, 1, 'getLicense has been called');
     assert.equal(callCounts.updateKeySession, 0, 'updateKeySession has not been called');
+    assert.equal(callCounts.licenseRequestAttempts, 0,
```

Comment:
Mentioned as a comment for the README, but since it's a bit odd to see `getLicense` has been called but `licenseRequestAttempts` is still at 0, may be worth saying in the expect something like `license request event not triggered (since no callback yet)`

Reply:
(Also for other test files with similar cases.)